proposals/dao/tip_121b.ts

```diff
@@ -0,0 +1,140 @@
+  console.log(' pcvStatsBefore.protocolEquity          [M]e18 ', Number(pcvStatsBefore.protocolEquity) / 1e24);
+  const pcvStatsAfter: PcvStats = await contracts.collateralizationOracle.pcvStats();
+  console.log('----------------------------------------------------');
+  console.log(' pcvStatsAfter.protocolControlledValue  [M]e18 ', Number(pcvStatsAfter.protocolControlledValue) / 1e24);
+  console.log(' pcvStatsAfter.userCirculatingFei       [M]e18 ', Number(pcvStatsAfter.userCirculatingFei) / 1e24);
+  console.log(' pcvStatsAfter.protocolEquity           [M]e18 ', Number(pcvStatsAfter.protocolEquity) / 1e24);
+  console.log('----------------------------------------------------');
+  const pcvDiff = pcvStatsAfter.protocolControlledValue.sub(pcvStatsBefore.protocolControlledValue);
+  const cFeiDiff = pcvStatsAfter.userCirculatingFei.sub(pcvStatsBefore.userCirculatingFei);
+  const eqDiff = pcvStatsAfter.protocolEquity.sub(pcvStatsBefore.protocolEquity);
+  console.log(' PCV diff                               [M]e18 ', Number(pcvDiff) / 1e24);
+  console.log(' Circ FEI diff                          [M]e18 ', Number(cFeiDiff) / 1e24);
+  console.log(' Equity diff                            [M]e18 ', Number(eqDiff) / 1e24);
+  console.log('----------------------------------------------------');
+
+  expect(eqDiff).to.be.bignumber.greaterThan(PCV_DIFF_LOWER);
+  expect(eqDiff).to.be.bignumber.lessThan(PCV_DIFF_UPPER);
+
+  // 1. Verify FEI burned as expected
+  const feiSupplyDiff = initialFeiSupply.sub(await contracts.fei.totalSupply());
+  expect(feiSupplyDiff).to.equal(TC_FEI_BALANCE);
+
+  // 2. Verify TRIBE sent to DAO treasury
+  const tcTribeBalanceDiff = (await contracts.tribe.balanceOf(addresses.core)).sub(initialTCTribeBalance);
+  expect(tcTribeBalanceDiff).to.equal(TC_TRIBE_BALANCE);
+
+  // 3. Verify TC has no funds
+  expect(await contracts.fei.balanceOf(addresses.tribalCouncilTimelock)).to.equal(0);
+  expect(await contracts.tribe.balanceOf(addresses.tribalCouncilTimelock)).to.equal(0);
+
+  // 4. Verify no FEI on DAO timelock
+  expect(await contracts.fei.balanceOf(addresses.feiDAOTimelock)).to.equal(0);
+
+  // 5. Verify all LUSD, WETH and DAI moved out of LBP swappers
+  // Verify nothing left on swappers
+  expect(await contracts.dai.balanceOf(addresses.lusdToDaiSwapper)).to.equal(0);
+  expect(await contracts.lusd.balanceOf(addresses.lusdToDaiSwapper)).to.equal(0);
+
+  expect(await contracts.weth.balanceOf(addresses.ethToDaiLBPSwapper)).to.equal(0);
+  expect(await contracts.dai.balanceOf(addresses.ethToDaiLBPSwapper)).to.equal(0);
+
+  // Verify LUSD Holding deposit empty
+  expect(await contracts.lusd.balanceOf(addresses.lusdHoldingPCVDeposit)).to.equal(0);
+
+  // Verify swapped tokens ended up at destinations correctly
+  expect(await contracts.dai.balanceOf(addresses.daiHoldingPCVDeposit)).to.be.bignumber.greaterThan(
+    MIN_LUSD_SWAP_DAI.add(MIN_WETH_SWAP_DAI) // DAI from WETH and LUSD swaps
+  );
```

Comment:
there's currently 58M in it and 34M + 19M will come from TC multisig `swap()` calls, so I think you should check a balance increase and not the absolute balance ?